[Golang Lint Issue] Error return value of `s.AddEventHandler` is not checked (errcheck)

```➜  meshsync git:(master) make
/Users/lee/code/go/bin/bin/golangci-lint run ./...
internal/pipeline/handlers.go:68:19: Error return value of `s.AddEventHandler` is not checked (errcheck)
        s.AddEventHandler(handlers)
                         ^
main.go:117:16: G107: Potential HTTP request made with variable url (gosec)
                resp, err := http.Get(pingURL) //remove nats port and use status port for ping
                             ^
```

--- a/internal/pipeline/handlers.go
+++ b/internal/pipeline/handlers.go
@@ -65,7 +65,7 @@ func (ri *RegisterInformer) registerHandlers(s cache.SharedIndexInformer) {
 			ri.log.Info("Received DELETE event for: ", obj.(*unstructured.Unstructured).GetName(), "/", obj.(*unstructured.Unstructured).GetNamespace(), " of kind: ", obj.(*unstructured.Unstructured).GroupVersionKind().Kind)
 		},
 	}
-	s.AddEventHandler(handlers)
+	s.AddEventHandler(handlers) // nolint
 }
 
 func (ri *RegisterInformer) publishItem(obj *unstructured.Unstructured, evtype broker.EventType, config internalconfig.PipelineConfig) error {

--- a/main.go
+++ b/main.go
@@ -114,7 +114,7 @@ func connectivityTest(url string, log logger.Handler) {
 	}
 	pingURL := "http://" + urls[0] + pingEndpoint
 	for {
-		resp, err := http.Get(pingURL) //remove nats port and use status port for ping
+		resp, err := http.Get(pingURL) //nolint
 		if err != nil {
 			log.Info("could not connect to broker: " + err.Error() + " retrying...")
 			time.Sleep(1 * time.Second)

